### ISSUE TITLE
"windows" as GRanges vs GRangesList

### ISSUE THREAD

**Issue by jwalabroad** [mskilab/gTrack#2]
I noticed that there is code to loop through "windows" in plot, where each windows is a GRangesList. What is the reason why there would be a GRangesList of windows, as opposed to an st-reduced GRanges specifying the coordinates to plot on? Can we just make these GRanges with no window loop?

**Comment by mskilab** [comment]
yeah this is an original that was built into gTrack / trackData at the beginning but we never use.  

providing a grangeslist windows of length 2 will create two vertically stacked views on the same gTrack object, with rearrangement links potentially between them.  It ended up looking very messy, so we never use it.  I haven't even tried it in a while.

You can try removing this - but it won't be straightforward.  The canvas partitioning, scaling, indexing of tracks and assigning their real estate on the screen depends on it.   

Best approach is to just require granges up front ie conceal this "feature" from the user.
